libs/dh/wholesale/feature-settlement-reports/src/lib/tabs/dh-wholesale-settlements-reports-tabs-balance.component.ts

```diff
@@ -237,4 +241,12 @@ export class DhWholesaleSettlementsReportsTabsBalanceComponent
           }),
       });
   }
+
+  private tryAddOneDay(value: unknown): unknown {
```

Comment:
Question: From what I can see, the value should be of type either `string | undefined`, so I can't see how your if statement is ever being executed?

Reply:
I can't use string or undefined.